lib/user.rb

```diff
@@ -0,0 +1,22 @@
+class User
+
+  attr_reader :name, :jokes
+  def initialize(name)
+    @name = name
+    @jokes = []
+  end
+
+  def learn(joke)
+    @jokes << joke
+  end
+
+  def tell(tellee, joke)
+    tellee.jokes << joke
```

Comment:
tellee 😆 

nice work with this-- you could also use the method that you just wrote that does this same work, since tellee is an instance of a user, you can call user methods on it. 

`tellee.learn(joke)`